modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-settings_{context}"]
+= Configuring the vSphere connection settings
+
+[role="_abstract"]
+Modify the following vSphere configuration settings:
+
+* vCenter network address
+* vCenter username
+* vCenter password
+* vCenter address
+* vSphere data center
+* vSphere datastore
+* Virtual machine of the cluster
+
+.Prerequisites
+* The {ai-full} has finished installing the cluster successfully.
+* The cluster is connected to link:https://console.redhat.com/.
+
+.Procedure
+. In the Administrator perspective, navigate to *Home > Overview*.
+. Under *Status*, click *vSphere connection*. This opens the *vSphere connection configuration* wizard.
+. In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *Username* field, enter your vSphere vCenter username. 
+. In the *Password* field, enter your vSphere vCenter password. 
++
+[WARNING]
+====
+The system will store the username and password into the `vsphere-creds` secret name in the  `kube-system` namespace of the cluster. An incorrect vCenter username or password will make the cluster nodes unschedulable.
+====
++
+. In the *Datacenter* field, enter the name of the vSphere data center that contains the virtual machines used to host the cluster; for example, `SDDC-Datacenter`.
+. In the *Default data store* field, enter the path and name of the vSphere data store that is to store the persistent data volumes; for example, `/SDDC-Datacenter/datastore/datastorename`.
++
+[WARNING]
+====
+Updating the vSphere data center or default data store once the configuration has been saved will detach any active vSphere `PersistentVolumes`. 
+====
++
+. In the *Virtual Machine Folder* field, enter the data center folder that contains the virtual machine of the cluster; for example, `/SDDC-datacenter/vm/ci-ln-hjg4vg2-c61657-t2gzr`. For the {product-title} installation to succeed, all virtual machines comprising the cluster must be located in a single data center folder.
```

Comment:
`/SDDC-datacenter/vm/ci-ln-hjg4vg2-c61657-t2gzr` better to change SDDC-datacenter to SDDC-Datacenter in consistent with above example

Reply:
@mareklibra Please could you address this?

Reply:
Back then, at the time of implementation, we were not considering multi-zones. I remember discussions happening afterwards.
@vrutkovs , do you know?

Reply:
Not aware of any multizone discussions. Its relatively new, so I suggest this document to mention single zone - and mention necessary changes in additional document